e2e/tests/settlement_without_onchain_liquidity.rs

```diff
@@ -156,7 +156,7 @@ async fn onchain_settlement_without_liquidity(web3: Web3) {
             web3: web3.clone(),
         }),
     );
-    let solver = solver::solver::naive_solver();
+    let solver = solver::solver::naive_solver(solver_account);
```

Comment:
I see an inconsistency here `solver_account` instead of solver from above. Maybe we could use this `solver_account` everywhere? or `solver_wallet`